lib/screens/classes/classes_screen.dart

```diff
@@ -0,0 +1,117 @@
+  Stream<List<Classes>> _classesStream;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  void initState() {
+    super.initState();
+    _classesStream = context.read<ClassesRepository>().getClasses();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(
+        scaffoldKey: _scaffoldKey,
+        title: Text('Schedule'),
+        shape: kAppBarShape,
+      ),
+      body: StreamBuilder<List<Classes>>(
+        stream: _classesStream,
+        builder: (BuildContext context, AsyncSnapshot<List<Classes>> snapshot) {
+          if (snapshot.hasData) {
+            final List<Classes> currentClass = snapshot.data;
+            final List<String> classHours =
+                currentClass[0].classAthletes.keys.toList();
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                DividerMedium(),
+                Padding(
+                  padding: const EdgeInsets.only(left: kDefaultPadding),
+                  child: Text(
+                    'Today\'s Schedule',
+                    style: TextStyle(fontSize: 24),
+                  ),
+                ),
+                DividerMedium(),
+                Expanded(
+                  child: ListView.separated(
+                    itemCount: currentClass[0].startingHours.length,
+                    itemBuilder: (context, index) {
+                      final String currentClassHour = classHours[index];
+                      return InkWell(
+                        onTap: () {},
+                        child: DefaultCard(
+                          child: Row(
+                            children: [
+                              Column(
+                                children: [
+                                  Text(
+                                    DateFormat.jm()
+                                        .format(currentClass[0]
+                                            .startingHours[index])
+                                        .toString()
+                                        .toLowerCase(),
+                                    style: TextStyle(
+                                        fontSize: 22, color: kWhiteTextColor),
+                                  ),
+                                  Text(DateFormat('E d, MMM')
+                                      .format(currentClass[0].classDate)
+                                      .toString()),
+                                ],
+                              ),
+                              Flexible(
+                                child: ListTile(
+                                  title: Text(
+                                    currentClass[0].startingHours[index] !=
+                                            DateTime.fromMillisecondsSinceEpoch(
+                                                1616065200000)
```

Comment:
Don't use Epoch to compare it with an hour. Epochs describes a date and time. If you only want to compare the time, it doesn't make sense to use the date also :(.

Try to think of a different way or search on the internet what's the norm on the inverted